Source/WebKit/UIProcess/WebProcessPool.h

```diff
@@ -624,6 +624,9 @@ class WebProcessPool final
     void registerAssetFonts(WebProcessProxy&);
 #endif
 
+    void setHasReceivedAXRequestInUIProcess() { m_hasReceivedAXRequestInUIProcess = true; }
```

Comment:
I think we usually use the `mark` prefix instead of `set` when there is no parameter like this.

Reply:
Thanks for reviewing! I will fix these issues in the upcoming patch.